src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -125,44 +134,68 @@ public bool GetWordVector(ref DvText word, float[] wordVector)
 
         private const int Timeout = 10 * 60 * 1000;
 
-        /// <summary>
-        /// Public constructor corresponding to <see cref="SignatureDataTransform"/>.
-        /// </summary>
-        public WordEmbeddingsTransform(IHostEnvironment env, Arguments args, IDataView input)
-                : base(env, RegistrationName, Contracts.CheckRef(args, nameof(args)).Column,
-                     input, TestIsTextVector)
+        public WordEmbeddingsTransform(IHostEnvironment env, string inputColumn, string outputColumn,
+           PretrainedModelKind? modelKind)
```

Comment:
>PretrainedModelKind? [](start = 11, length = 20)

Nullable type looks weird here.
In argument/columns it's used to setup default value to columns where it's optional, but tend to flatten our structure with ColumnInfo class in each Transformer.

Can you also use ColumnInfo class instead of reusing Column class (Column is mostly for arguments and command line)